### PR TITLE
bug 1452681 - artefact 2

### PR DESCRIPTION
**Pull request by willkg**
Example output:

```
$ PIGEON_BUILD_ID=foo ./bin/build_artifact.sh && more version.json 
{"commit":"50d499d61b2a18826590299366125aeffd6e8bcb","version":"","source":"https://github.com/mozilla-services/socorro-pigeon","build":"foo"}
```

### CODE REVIEW

File: bin/build_artifact.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 SHA1="$(git rev-parse HEAD)"
 TAG=""
-PROJECT_REMOTE="$(git remote -v | grep fetch | head -n 1 | awk '{print $2}')"
-BUILD=""
+PROJECT_REMOTE="https://github.com/$(git remote | head -n 1 | xargs git remote get-url | sed -E 's/[^:]*:(.*)\..*$/\1/')"
```
Comment (willkg) [inline bin/build_artifact.sh]: This assumes the first remote listed is the one you want. I have no idea what the box that does the deploys looks like, so I don't know if this is unreasonable or not.

Comment (milescrabill) [reply bin/build_artifact.sh]: This produced:

```
{"commit":"683978fe97a311f8cdc330295e40567f916f78a6","version":"","source":"https://github.com///github.com/mozilla-services/socorro-pigeon","build":"nobuild"}
```

on my local machine with remotes:

```
origin	git://github.com/mozilla-services/socorro-pigeon.git (fetch)
origin	git://github.com/mozilla-services/socorro-pigeon.git (push)
willkg	git@github.com:willkg/socorro-pigeon.git (fetch)
willkg	git@github.com:willkg/socorro-pigeon.git (push)
```

So naturally I took the easy fix: `git remote set-url origin git@github.com:mozilla-services/socorro-pigeon.git`. I think this should be fine.

Comment (milescrabill) [reply bin/build_artifact.sh]: So, the way I'm checking out pigeon with ansible is what I described in the previous comment. So we should fix this.

Comment (milescrabill) [reply bin/build_artifact.sh]: I tested and `git remote get-url origin | sed 's/^.*\/\///'` should be fine for the way we check out the pigeon repo. (even though it's not fine in this case for the `willkg` remote I created locally to test this PR.)

Comment (willkg) [reply bin/build_artifact.sh]: How about you add `PIGEON_SOURCE` to the environment?

Comment (willkg) [reply bin/build_artifact.sh]: Er, let me extrapolate on that. How about we do the same thing with source that we did with build where you set `PIGEON_SOURCE` in the environment and this script picks that up and uses it as the source value?

Comment (milescrabill) [reply bin/build_artifact.sh]: My PR: https://github.com/mozilla-services/cloudops-deployment/pull/1833.

Waiting on this to be merged to merge mine.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**